packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -2254,7 +2255,8 @@ export class ProjectService extends BaseService {
     > & { warehouseType: WarehouseTypes }): string {
         if (!indexColumn) throw new ParameterError('Index column is required');
         const q = getFieldQuoteChar(warehouseType);
-        const userSql = sql.replace(/;\s*$/, '');
```

Comment:
We don't need to change the logic in applyPivotToSqlQuery.

Reply:
I added this since without it warehouses that don't support // will throw an error when we pivot. 

Removing these comments is a bit of a pre-processing step we are supporting. We *could* try to only support it in warehouses that support // and we don't need this here. But I'm not sure how to reconcile the UX of entering SQL with unsupported comment types depending on the warehouse.